### DISCUSSION TITLE
Persisted Query support

### DISCUSSION THREAD

**Discussion by jlucaso1** [0no-co/gql.tada#67] [Ideas]
My team's only obstacle to migrating to `graphql.tada` is support for persisted queries. Is there any way to use and maintain type checking?

References:
[Mercurius Persisted Query documentation](https://github.com/mercurius-js/mercurius/blob/master/docs/persisted-queries.md)

**Comment by kitten** [comment]
We're coincidentally working on an RFC and plan for this right now, so we'll have an answer to this hopefully very soon, if not within days ❤️

**Comment by maraisr** [comment]
Just noting here, we are also so excited about persisted queries. go team-🎉
